charts/devnet/templates/chains/cosmos/genesis.yaml

```diff
@@ -51,6 +51,14 @@ spec:
     spec:
       {{- include "imagePullSecrets" $chain | indent 6 }}
       initContainers:
+        {{- if hasKey $chain "storage" }}
+        - name: init-chmod-data
+          image: busybox
+          command: ['sh', '-c', 'chmod -R 0777 {{ $chain.home }}']
```

Comment:
hmm maybe there is another way to do this. Will have to look into this.

Reply:
I think it depends on what user runs the containers. If you know a better way, I would gladly implement it.